contracts/modules/LoanOpenings.sol

```diff
@@ -412,25 +421,25 @@ contract LoanOpenings is
                 loanParamsLocal,
                 loanLocal,
                 initialMargin,
-                sentValues[4],
+                sentValues.collateralTokenSent,
                 collateralAmountRequired
             ),
             "collateral insufficient"
         );
 
-        loanLocal.collateral = loanLocal.collateral.add(sentValues[4]);
+        loanLocal.collateral = loanLocal.collateral.add(sentValues.collateralTokenSent);
 
         if (isTorqueLoan) {
             /// reclaiming varaible -> interestDuration
```

Comment:
```suggestion
            /// reclaiming variable -> interestDuration
```
the code below is a great confusion
pls consider adding another variable for TorqueLoan and amend the comment above respectively